src/main/java/net/java/sen/tokenizers/ja/JapaneseTokenizer.java

```diff
@@ -28,6 +28,8 @@
 import net.java.sen.dictionary.Tokenizer;
 import net.java.sen.trie.CharIterator;
 
+import static java.lang.Character.UnicodeBlock.*;
```

Comment:
Could you please replace "*" to exact method names?

Reply:
I have pushed the updated version that avoids using "*".